### ISSUE TITLE
Pages exclusive for students are not tested for accessibility 

### ISSUE THREAD

**Issue by sbelsk** [Submitty/Submitty#6790]
**What problem are you trying to solve with Submitty**
Currently, the accessibility test/html validator is being done only from the instructor account on the sample course, so pages available to only students (like the student pages for polls, office hours queue, rainbow grades) or not being tested.

**Describe the way you'd like to solve this problem**
The tests should be modified to validate all the Submitty pages available through a student account in addition to the instructor account.
